packages/hoprd/src/api/v3/paths/tickets/price.ts

```diff
@@ -0,0 +1,74 @@
+import type { Operation } from 'express-openapi'
+import { STATUS_CODES } from '../../utils.js'
+import { debug, Hopr } from '@hoprnet/hopr-utils'
+
+const GET: Operation = [
+  async (req, res, _next) => {
+    const log = debug('hoprd:api:v3:get-ticket-price')
+    const { node }: { node: Hopr } = req.context
+
+    try {
+      const ticket_price = await node.getTicketPrice()
+      log(`retrieved ticket price ${ticket_price}`)
+
+      if (ticket_price === null) {
+        return res
+          .status(206)
```

Comment:
2XX is a success code. Please use a different value, e.g. https://en.wikipedia.org/wiki/HTTP_404